### PR TITLE
Add support for cocoapods_version

### PR DESCRIPTION
**Pull request by DanielTomlinson**
This pull request adds support for specifying a CocoaPods version inside the podspec (https://github.com/cocoapods/core/issues/240)

### CODE REVIEW

File: spec/specification_spec.rb [hunk 1]
```diff
@@ -93,6 +93,24 @@ module Pod
...
+
+        it 'passes when the requirement is satisfied' do
+          spec_1 = Specification.new { |s| s.cocoapods_version = '>= 0.1.0' }
+          should.not.raise { spec_1.validate_cocoapods_version }
+        end
+
+        it 'fails when the requirement is not satisfied' do
+          spec_1 = Specification.new { |s| s.cocoapods_version = '= 999999.0.0' }
+          should.raise(Informative) { spec_1.validate_cocoapods_version }.message.
+            should.match //
```
Comment (kylef) [inline spec/specification_spec.rb]:
```ruby
should.match /CocoaPods version/
```

**Comment by DanielTomlinson** [comment]
@kylef Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # CocoaPods Core Changelog
 
+## Master
+
+##### Enhancements
+
+* Add support for specifying the CocoaPods version in the podspec.
+	[Daniel Tomlinson](https://github.com/DanielTomlinson)
```
Comment (segiddins) [inline CHANGELOG.md]: spacing

**Comment by segiddins** [comment]
:+1: 

:eyes: @kylef ?

**Comment by kylef** [comment]
:shipit:

**Pull request merged**